source/_integrations/matter.markdown

```diff
@@ -255,6 +255,32 @@ Follow these steps if you have added a Matter device to Home Assistant and you w
    - For example, if you shared it with Google Home, it lists **Google LLC**.
      ![image](/images/integrations/matter/matter_share-device-with-other-platform.png)
 
+## About Matter device information
+
+The device information section provides some networking information of a device. For devices that are behind a bridge, this information is not available.
```

Comment:
```suggestion
The device information section provides some diagnostic information of a device.
```

I'd move this more to the back: For devices that are behind a bridge, this information is not available.

Reply:
Well, if it is stated here, I know that I can stop reading if I have a bridged device